.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+**Bug Report**
+
+A clear and concise description of what the bug is.
```

Comment:
What is the minimum set of factors in the description that can make it 'clear and concise'?

Reply:
Depends on the case... that's why it is stated as it is. Any suggestions you believe are a must for every case?